index.html

```diff
@@ -706,11 +706,11 @@
 contested ([[?Privacy-Contested]]). This makes privacy a problem of collective action ([[?GKC-Privacy]]).
 Group-level [=data processing=] may impact populations or individuals, including in
 ways that [=people=] could not control even under the optimistic assumptions of [=consent=].
-For example, based on group-level analysis, a company may know that <var>site.example</var>
-is predominantly visited by [=people=] of a given race or gender, and decide not to run its
-job ads there. Visitors to that page are implicitly having their [=data=] processed in
-[=inappropriate=] ways, with no way to discover the discrimination or seek relief
-([[?Relational-Governance]]).
+For example, based on group-level analysis, a company may know that <var>Niche
+Magazine</var> or <var>site.example</var> is predominantly read by [=people=]
+of a given race or gender, and decide not to run its job ads there. Readers of
+those publications are implicitly having their [=data=] processed in
+[=inappropriate=] ways ([[?Relational-Governance]]).
```

Comment:
Is it worth losing `, with no way to discover the discrimination or seek relief`?